src/styles/blocks.scss

```
@@ -0,0 +1,64 @@
+* {
+  box-sizing: border-box;
+}
+
+@each $name, $index in $dayOfWeek {
+  .calendar__start-day--#{$name}
+    .calendar__day:first-child {
+    margin-left: calc((#{$squareBox} + #{$gap}) * (#{$index} - 1));
+    }
+}
+
+@for $days from 28 through $monthLength {
+  .calendar__month-length--#{$days}
+    .calendar__day:nth-child(n + #{$days + 1}) {
+    display: none;
+    }
+}
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  width: calc(
+    (#{$squareBox} * #{$dayInWeek})
+    + ((#{$dayInWeek} * #{$gap}) - #{$gap})
+    + (#{$paddingCalendar * 2})
+  );
+  padding: $paddingCalendar;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    width: $squareBox;
+    height: $squareBox;
+    border: $borderWidth solid #000;
+    background-color: #eee;
+    transition: transform 0.5s, background-color 0.5s;
```

You can use the keyword "all" here. But if you want to leave it as it is, then transfer the transition delay to a variable, since it is repeated.
```suggestion
    transition: all 0.5s;
```